CHANGELOG.md

```diff
@@ -48,5 +48,210 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Docs
 
```

Comment:
Please update the latest changelog of this branch.
The changeslogs of `v2.0.1` and `v2.0.2` are not updated.